### DISCUSSION TITLE
this thing is marked as malware

### DISCUSSION THREAD

**Discussion by Petrosmine** [mircokroon/minecraft-world-downloader#571] [General]
can you can see it yourself https://www.virustotal.com/gui/file/b14439a20d903be5c1164d368287ee2a6749303b9c9f8664b88b00e02c02af15

**Comment by mircokroon** [comment]
False-positive, you can check the source code yourself and verify the build matches the binaries provided in the release.

**Comment by rautamiekka** [comment]
> can you can see it yourself https://www.virustotal.com/gui/file/b14439a20d903be5c1164d368287ee2a6749303b9c9f8664b88b00e02c02af15

Just a single detection and an AI one at that, doesn't mean much anything.
